app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ProjectsController < ApplicationController
+
+  def show
+    @project = Project.find(params[:id])
+  end
+end
+
+
+private
+
+def project_params
+    params.permit(:name, :material)
+end
```

Comment:
Your show action looks good. I don't think you need the `project_params` right now since you don't have a `create` or `update` action/functionality at this time. We can even see that `project_params` isn't called anywhere in this class.